### PR TITLE
PSR-12

### PR DESCRIPTION
**Pull request by nezaniel**
This adds PSR-12 compatibility to the Neos code base

### CODE REVIEW

**Comment by nezaniel** [comment]
> nice how can one help? or is it done ^^

well, it's done, but now that the tests are green again, it's open for review :)

File: Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/Feature/NodeRenaming.php [hunk 1]
```diff
@@ -31,10 +32,12 @@ trait NodeRenaming
     public function whenNodeAggregateNameWasChanged(NodeAggregateNameWasChanged $event): void
     {
         $this->transactional(function () use ($event) {
-            foreach ($this->getProjectionHyperGraph()->findNodeRecordsForNodeAggregate(
-                $event->getContentStreamIdentifier(),
-                $event->getNodeAggregateIdentifier()
-            ) as $originNode) {
+            foreach (
+                $this->getProjectionHyperGraph()->findNodeRecordsForNodeAggregate(
+                    $event->getContentStreamIdentifier(),
```
Comment (mhsdesign) [inline Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/Feature/NodeRenaming.php:37]: i guess these `NodeAggregateTypeWasChanged` getters can be switched to property access too - ... sometime

File: Neos.ContentRepositoryRegistry/Configuration/Objects.yaml [hunk 2]
```diff
@@ -42,6 +42,11 @@ Neos\ContentRepository\Feature\ContentStreamCommandHandler:
 Neos\ContentRepositoryRegistry\Legacy\ObjectFactories\DbalContentGraphObjectFactory:
   scope: singleton
 
+Neos\ContentGraph\DoctrineDbalAdapter\Domain\Repository\ContentGraph:
```
Comment (mhsdesign) [inline Neos.ContentRepositoryRegistry/Configuration/Objects.yaml:45]: This seems to be the only thing added sofar ...

**Review by mhsdesign — APPROVED** [review]
Noice by reading

File: Neos.ContentRepositoryRegistry/Configuration/Objects.yaml (hunk 2 above)
Comment (mhsdesign) [reply Neos.ContentRepositoryRegistry/Configuration/Objects.yaml:45]: im just wondering why in this PR?

File: Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/Feature/NodeRenaming.php (hunk 1 above)
Comment (nezaniel) [reply Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/Feature/NodeRenaming.php:37]: yep, will be done when NodeMove is touched next time

File: Neos.ContentRepositoryRegistry/Configuration/Objects.yaml (hunk 2 above)
Comment (nezaniel) [reply Neos.ContentRepositoryRegistry/Configuration/Objects.yaml:45]: basically because otherwise it won't work ^^

**Review by skurfuerst — APPROVED** [review]

**Pull request merged**